### ISSUE TITLE
How to get start with VSRTorch

### ISSUE THREAD

**Issue by AIRedWood** [LoSealL/VideoSuperResolution#58]
Hello, I recently found out that your project has added the VSRtorch module. How can I use the VSRtorch training model? For example, I want to train msrn. How do I enter an instruction?

**Comment by LoSealL** [comment]
There's a brief introduction under `VSRTorch/Readme.md`. Most arguments are similar to tensorflow's, while the major difference is where I split `run.py` into `train.py` and `eval.py` and replace `--model` as an inplace argument.

**P.S** using `--cuda` to run with GPU.

**Comment by AIRedWood** [comment]
I have used vSRtorch to train on v-espcn. I recently learned about pytorch. The syntax of pytorch is much simpler than tf. Hahaha

**Comment by AIRedWood** [comment]
| 2019-06-03 05:36:43 | Epoch: 36/1000 | LR: 0.0001 |
100%|#####################################| 200/200 [00:11<00:00, 17.89batch/s, image=00.00319, flow=15.47342, tv=07.22975]
| Epoch average image = 0.002690 |
| Epoch average flow = 18.848642 |
| Epoch average tv = 7.170183 |
2019-06-03 05:36:54,282 WARNING: frames is empty. [size=10]
Test: 0it [00:00, ?it/s]

**Comment by AIRedWood** [comment]
However, what do these training messages mean? These are not the same as the loss shown by tf.

**Comment by AIRedWood** [comment]
| 2019-06-03 05:39:53 | Epoch: 174/1000 | LR: 0.0001 |
100%|######################################| 200/200 [00:12<00:00, 28.72batch/s, l2=74.15609, loss=392.70578, me=318.54968]
| Epoch average l2 = 168.231995 |
| Epoch average loss = 654.335754 |
| Epoch average me = 486.103790 |
WARNING:tensorflow:frames is empty. [size=4]
Test: 0it [00:00, ?it/s]

**Comment by AIRedWood** [comment]
These are the information displayed by tf

**Comment by LoSealL** [comment]
First of all, the validation data seems broken, no data was loaded.

Secondly, metrics words are defined in each model file. I dont insist to make both versions exactly the same.
